lib/calib/calib_basic.c

```diff
@@ -50,7 +50,7 @@ ATCA_STATUS calib_wakeup_i2c(ATCADevice device)
 
         do
         {
-            if (100000u < ATCA_IFACECFG_VALUE(iface->mIfaceCFG, atcai2c.baud))
+            if (100000u <= ATCA_IFACECFG_VALUE(iface->mIfaceCFG, atcai2c.baud))
```

Comment:
do we actually want to perform the ATACA_HAL_CHANGE_BAUD operation if the baud rate is already 100Kb, or do we just want to do the ATCA_HAL_CONTROL_WAKE?